### PR TITLE
No longer use python-distutils-extra

### LINKED ISSUE #853: Look into replacing python-distutils-extra
When trying to introduce automatic update checking for the Flatpak build, I've been told that [intltool was dead](https://github.com/flathub/shared-modules/pull/245#issuecomment-1518719041). Maybe it's worth looking into alternatives before it breaks beneath us, not a high priority though.

According to https://wiki.gnome.org/Projects/GnomeCommon/Migration we should be able to replace it with gettext:

> intltoolize is obsolete: gettext 0.19.7 includes all the functionality GNOME projects used from intltool, so you should port from intltool to gettext.

### PR DESCRIPTION
**Pull request by jeromerobert**
* python-distutils-extra was added by 2077a73
* it relies on intltool which is deprecated as gettext can now do the job alone
* intltool relies on perl which is a tricky dependency on Windows
* PyPA has announced disruptive changes to distutils/setuptools/pip that will propably break python-distutils-extra
* clean_i18n is removed as it was not very useful
* Close #853

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -20,9 +20,18 @@
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 #
 
+from setuptools import Command
 from setuptools import setup
-from DistUtilsExtra.command import (
-    build_i18n, clean_i18n, build_extra, build_icons)
+# for the time being do not use setuptools.command because:
+# * setuptools is to old in many distro (ex: Ubuntu 22.04)
+# * The Fedora package relies on wheel so it would not contains translation and
```
Comment (dreua) [inline setup.py:27]: Do you want me to look after the Fedora stuff? Pretty sure we can get rid of the wheel. Wheel's not ideal anyway ;) (It was just the easiest back when I first packaged this.)

Comment (jeromerobert) [reply setup.py:27]: You read my mind. Yes !

**Comment by jeromerobert** [comment]
I cannot test the Windows installer of this PR for now. I'll do it as soon as I have a Windows box. Please do not merge until Windows installer has been tested.

**Comment by dreua** [comment]
More discussion: https://github.com/pdfarranger/pdfarranger/discussions/860

**Comment by dreua** [comment]
Test results on Windows 10, msi and zip from https://github.com/pdfarranger/pdfarranger/actions/runs/4903360382?pr=859:

1. Yeah, dark mode gets recognized and looks super cool 😎
2. Looks all good, manual language change works
4. You need to click through "Windows protected your PC" -> "More info" -> "Run anyway" (once, either during installation or first run of the portable version)

All above is true for both portable zip and installed msi.

File: setup.py (hunk 1 above)
Comment (dreua) [reply setup.py:27]: Just to be on the same page: The current version still works with the wheel but you want me to get rid of it in order to further improve the thing, correct?

**Pull request merged**